### PR TITLE
fix the icon spacing on mobile

### PR DESCRIPTION
**Pull request by nandiheath**

### CODE REVIEW

File: core/src/components/Atom/SocialBrands.js [hunk 1]
```diff
@@ -14,6 +14,9 @@ const StyledContainer = styled(Box)`&& {
 
 const StyledLink = styled(OutboundLink)`&& {
   margin: 0 16px;
+  @media screen and (max-width: 400px) {
```
Comment (mingchuno) [inline core/src/components/Atom/SocialBrands.js]: seems we don't need hardcode here?
https://material-ui.com/customization/breakpoints/
@caseykwok

Comment (gaplo917) [reply core/src/components/Atom/SocialBrands.js]: should use bps.only('xs')

Comment (nandiheath) [reply core/src/components/Atom/SocialBrands.js]: never use these before. how to integrate them with styled components?

Comment (mingchuno) [reply core/src/components/Atom/SocialBrands.js]: @nandiheath sth like this
`import {bps} from '../../utilis/muiTheme';`

**Pull request merged**